.github/workflows/main.yml

```diff
@@ -10,7 +10,9 @@ on:
     branches: [ develop ]
     paths-ignore:
       - 'doc/**'
-
+  # Allows you to run this workflow manually from the Actions tab
```

Comment:
The comment should include the reasoning behind adding this rather than explaining what it means.